apps/vite/src/components/AddEventModal.tsx

```
@@ -1,99 +1,101 @@
+import { apiBaseUrl } from '../helper/api';
+import DropDown from './DropDown';
+import NepaliDatePicker from './NepaliDatePicker';
+import Spinner from './Spinner';
 
 function getCombinedDateTime(date: Date, time: string) {
-  const timeParts = time.split(":")
-  date.setHours(parseInt(timeParts[0] ?? "", 10))
-  date.setMinutes(parseInt(timeParts[1] ?? "", 10))
-  return date.toISOString()
+  const timeParts = time.split(':');
+  date.setHours(parseInt(timeParts[0] ?? '', 10));
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)